### ISSUE TITLE
Name object in `write_topojson()` something besides `foo`?

### ISSUE THREAD

**Issue by josiekre** [ropensci/geojsonio#129]
How does one get a topojson file out with an object named something besides `foo`?

```
{"type":"Topology","objects":{"foo":{"type":"GeometryCollection","geometries":...
```

**Comment by sckott** [comment]
hi @josiekre 👋 thanks for this. Are you referring to the function `geojsonio::topojson_write` ?

also, what versino of the package do you have?

**Comment by josiekre** [comment]
Yes that's the function. Sorry about that. I updated the issue title. (The `tidyverse` world made me think it would be the other way around.)

I'm running `v0.5.0` at the moment.

**Comment by sckott** [comment]
we do that here https://github.com/ropensci/geojsonio/blob/master/R/geo_topo.R#L76 

ill see about letting user specify that key

**Comment by josiekre** [comment]
That's where it is! I could not find the right R function that was actually setting that.

**Comment by sckott** [comment]
@josiekre try again after reinstalling `devtools::install_github("ropensci/geojsonio@topojson-object-name")`

```r
vec <- c(-99.74, 32.45)
x <- topojson_write(vec, object_name = "California")
readLines(x$path)
```

**Comment by josiekre** [comment]
This works! I tested it with my code too. 

When is your next release to CRAN planned? I will update my DESCRIPTION file to install this branch until it is merged and released on CRAN.

**Comment by sckott** [comment]
Great. Glad it works. Next milestone https://github.com/ropensci/geojsonio/milestone/11 hopefully end of next week
